### ISSUE TITLE
ggplot graphs do not appear with R 3.4.2

### ISSUE THREAD

**Issue by dannyparsons** [jmp75/rdotnet#59]
I have just updated to R 3.4.2 from 3.4.1 and now when I generate a `ggplot` graph e.g.

`r.Evaluate("ggplot(mpg, aes(displ, hwy, colour = class)) + geom_point()")`

they no longer appear in an R viewer (or anywhere in my application). There is no error message in the Visual Studio Output window.

Non `ggplot` graphs e.g. `r.Evaluate("plot(1:10)")` works as before and produces the graph in an R viewer.

The only way I have been able to produce `ggplot` graphs in RDotNet with R 3.4.2 is to do:

`r.Evaluate("ggplot2:::print.ggplot(ggplot(mpg, aes(displ, hwy, colour = class)) + geom_point())")`

I tried reverting to previous versions of the `ggplot2` package but I still have the same issue. However, if I uninstall R 3.4.2 then ggplot graphs work as before.

Any idea of the cause of this? I looked in the [R News](https://cran.r-project.org/doc/manuals/r-devel/NEWS.html) but couldn't see any obvious change in R 3.4.2 which would affect this.

**Comment by lrasmus** [comment]
Curious if things work in 3.4.1 or not?  We've noticed some issues as well in 3.4.2 and 3.4.3 (similar to what's reported https://github.com/jmp75/rdotnet/issues/62).  Haven't gotten to the bottom of it yet, but appreciate you reporting it so I know we're not totally off base!

**Comment by dannyparsons** [comment]
Yes, it does sound like it's caused by small changes between versions of R but I haven't noticed anything in the documentation that looked related to this. Will let you know if I find out any more.

**Comment by gkapellmann** [comment]
Is there any more info regarding this issue? At the moment we are in R4.2.2 but the ggplot2 doesnt show anything, and it can even block the program.

I have tried different versions of:
`engine.Evaluate("myChart <- ggplot(my_data, aes(x=Price, y=Quantity)) + geom_line()"); 
 engine.Evaluate("print(myChart)");`
 
But no luck.
